.github/workflows/ci.yml

```
@@ -67,3 +67,77 @@ jobs:
 
       - name: Run unit tests
         run: pnpm test:unit
+
+  e2e:
+    if: ${{ hashFiles('playwright.config.ts') != '' }}
```

actionlint (1.7.12)

[error] calling function "hashFiles" is not allowed here. "hashFiles" is only available in "jobs.<job_id>.steps.continue-on-error", "jobs.<job_id>.steps.env", "jobs.<job_id>.steps.if", "jobs.<job_id>.steps.name", "jobs.<job_id>.steps.run", "jobs.<job_id>.steps.timeout-minutes", "jobs.<job_id>.steps.with", "jobs.<job_id>.steps.working-directory". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)